src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRouteNames.cs

```diff
@@ -56,6 +56,11 @@ internal class KnownRouteNames
     internal const string VersionedRetrieveWorkitemInstance = nameof(VersionedRetrieveWorkitemInstance);
     internal const string RetrieveWorkitemInstance = nameof(RetrieveWorkitemInstance);
 
+    internal const string VersionedPartitionChangeStateWorkitemInstance = nameof(VersionedPartitionChangeStateWorkitemInstance);
+    internal const string PartitionedChangeStateWorkitemInstance = nameof(PartitionedChangeStateWorkitemInstance);
```

Comment:
If Ali merges his version PR, we need to remove two of these

Reply:
I don't think we want to remove these. These are explicitly for "Update Workitem State" action. Ali's is "Update Workitem", which has different route (as it includes TransactionUID also).